### ISSUE TITLE
Possibly need to increase file descriptors

### ISSUE THREAD

**Issue by thieman** [gamechanger/dusty#617]
Report from @aehsu. She encountered this line during a `dusty up`, after which Dusty became unresponsive:

```
Stopping dusty_coreRedis_1... done
Error: [Errno 24] Too many open files
Stopping dusty_zookeeper_1... done
```

It's not clear to me whether this would be coming from the Mac or if Compose is bubbling it up from the VM, but I think it's worth investigating to see if there are some easy fixes we could put in place on the VM.

**Comment by aehsu** [comment]
Error log

2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/threading", line 810, in __bootstrap_inner
2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/threading", line 763, in run
2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/compose.utils", line 31, in inner_execute_function
2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/compose.project", line 203, in <lambda>
2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/compose.container", line 143, in stop
2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/docker.utils.decorators", line 20, in wrapped
2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/docker.client", line 815, in stop
2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/docker.clientbase", line 83, in _post
2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/requests.sessions", line 508, in post
2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/requests.sessions", line 465, in request
2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/requests.sessions", line 573, in send
2016-01-13 13:06:46 INFO:socket_logger File "/compose/build/docker-compose/out00-PYZ.pyz/requests.adapters", line 415, in send
2016-01-13 13:06:46 INFO:socket_logger ConnectionError: ('Connection aborted.', error(24, 'Too many open files'))

**Comment by thieman** [comment]
I'm not sure it's possible for us to do this ourselves, the user would need to set a higher open files limit themselves using `ulimit -n 10240` or a more robust, permanent solution like one listed [here](http://superuser.com/questions/302754/increase-the-maximum-number-of-open-file-descriptors-in-snow-leopard). Closing as I don't think this is something we can fix in Dusty.

**Comment by thieman** [comment]
Trying to fix this again in #660
